src/app/modules/tld-extensions/extensions/function.extensions.ts

```diff
@@ -1,10 +1,11 @@
-interface Function {
-    callDelayed(previousThis: any, ...args: any): any;
-    callDelayedTimout(previousThis: any, timeout: number, ...args: any): any;
-    applyDelayed(previousThis: any, args?: any[]): any;
-    applyDelayedTimout(previousThis: any, timeout: number, args?: any[]): any;
+declare global {
+    interface Function {
+        callDelayed(previousThis: any, ...args: any): any;
```

Comment:
this ...args not supporting typescript 2.x which means that it's not supporting angular <= 6.x